docs/03-endpoints/api-admin/projects.md

```diff
@@ -814,6 +815,49 @@ Example response:
 }
 ```
 
+### Set Restricted View Settings
+
+Both routes take String parameter which sets restricted view size in one of two formats: as an image dimensions or a 
+percentage. The dimensions pattern looks like: `!X,X`, where X is the number representing scaled image dimensions in
+a square, so that the width and height of the returned image are not greater than the requested value.
+Example: `!512,512` means the image's bigger side will be set to 512 pixels, setting the other side respectively to
+image aspect ratio. The percentage pattern looks like: `pct:X`, where X is the number between 1-100 representing the
+percentage the image will be scaled to. Example: `pct:1` means the image will be scaled to 1% of the original image
+size. **Both routes are implemented in `ZIO-HTTP`, thus can be reached only under port `5555`.**
+
+Permissions: ProjectAdmin/SystemAdmin
+
+Request definition:
+- `POST /admin/projects/iri/{iri}/RestrictedViewSettings`
+- `POST /admin/projects/shortcode/{shortcode}/RestrictedViewSettings`
+
+Description: Set the project's restricted view
+
+Required payload:
+- `size`
+
+Example request:
+
+```bash
+curl --request POST 'http://0.0.0.0:5555/admin/projects/iri/http%3A%2F%2Frdfh.ch%2Fprojects%2F0001/RestrictedViewSettings' \
+--header 'Authorization: Basic cm9vdEBleGFtcGxlLmNvbTp0ZXN0' \
+--data '{"size": "!512,512"}
+```
+
+```bash
+curl --request POST 'http://0.0.0.0:5555/admin/projects/shortcode/0001/RestrictedViewSettings' \
+--header 'Authorization: Basic cm9vdEBleGFtcGxlLmNvbTp0ZXN0' \
+--data '{"size": "!512,512"}
+```
+
+Example response:
+
+```jsonc
```

Comment:
I would have used `json`. I guess `jsonc` stands for json with comments, but you are not using comments.